infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/secure/jwtproxy/JwtProxyProvisioner.java

```diff
@@ -77,6 +81,7 @@
 
   static final String JWT_PROXY_CONFIG_FOLDER = "/config";
   static final String JWT_PROXY_PUBLIC_KEY_FILE = "mykey.pub";
+  static final String UNSECURED_PATHS_ATTRIBUTE = "unsecuredPaths";
```

Comment:
It would be better to move this constant to https://github.com/eclipse/che/blob/efaef7e0d1a05c03a6819f240a267a015a542f05/core/che-core-api-model/src/main/java/org/eclipse/che/api/core/model/workspace/config/ServerConfig.java#L37
and document it there.